demos/demo-minimal-js/index.js

```diff
@@ -3,10 +3,12 @@ import { errorType, callEndStatus } from "../../src/Constants";
 // import CallingExtensions, { Constants } from "@hubspot/calling-extensions-sdk";
 // const { errorType, callEndStatus } = Constants;
 
-const state = {
+export const state = {
```

Comment:
Expose state to the window object so that we can change any of the properties such as `fromNumber` before sending an event in the Demo Widget. For our acceptance tests, we should create an engagement state and call state to display in the UI like how we are doing so in the Mock UI.

Reply:
I would export it with a more definitive name. state is too generic. Thoughts?

Reply:
Sure, do you have any suggestions for the name? This is only exported in the `demo-minimal-js` iframe so we won't be seeing a conflict with any variables in the parent window

Reply:
We could also implement getters and setters here

Reply:
If the scope is minimal, i am fine. But, are we making updates to state properties? If yes, then it would be better if we have function to return initial state. So, we dont have accidental leaks.

Reply:
Yeah it's a minimal scope right now. Since we're not persisting the state across mounts, any changes to state properties would only affect the demo minimal js widget until we unmount the component.